charts/prometheus-kafka-exporter/templates/deployment.yaml

```diff
@@ -80,10 +80,9 @@ spec:
             - {{ . }}
           {{- end }}
           {{- end }}
+          {{- with .Values.env }}
           env:
```

Comment:
This line would disable the whole field if `.Values.env` were null. We would not like to do that since some variables are already defined in the template. One way would be starting below the line:

```suggestion
          env:
          {{- with .Values.env }}
```

Reply:
Good point, sorry I completely missed the other two variables. I've updated the PR accordingly and squashed the commits.